frontend/packages/client/src/const/index.js

```diff
@@ -3,6 +3,8 @@ export const MAX_FILE_SIZE = 5242880;
 // 1024 * 1024 * 2 = 2MB
 export const MAX_AVATAR_FILE_SIZE = 2097152;
 
+export const MAX_PROPOSAL_IMAGE_FILE_SIZE = 2097152;
+
```

Comment:
this is kind of repeated but applies only for in-line images updated to proposals